cores/esp8266/Esp.h

```diff
@@ -131,6 +133,13 @@ class EspClass {
         bool eraseConfig();
 
         inline uint32_t getCycleCount();
+
+        /**
+         * Be very careful in your callback.  Don't call anything that could
+         *  yield (so pretty much don't call Arduino code).  And god help you if
+         *  it crashes.
+         */
+        void installCustomCrashCallback( CrashCallback );
```

Comment:
I didn't know about that feature, that's way cleaner.  I'll make that change, thanks!